examples/ex28p.cpp

```diff
@@ -86,6 +86,8 @@ int main(int argc, char *argv[])
    MPI_Init(&argc, &argv);
    MPI_Comm_size(MPI_COMM_WORLD, &num_procs);
    MPI_Comm_rank(MPI_COMM_WORLD, &myid);
+   // HYPRE_Init();
+   // HYPRE_SetSpGemmUseCusparse(0);
```

Comment:
I think we should remove these, see https://github.com/mfem/mfem/pull/1492/files?file-filters%5B%5D=.cpp&file-filters%5B%5D=.hpp&file-filters%5B%5D=No+extension#r675263259

Reply:
Resolved by c754730